src/umockdev-record.vala

```diff
@@ -417,7 +428,41 @@ main (string[] args)
     foreach (string s in opt_evemu_events)
         record_script(s, "evemu");
 
-    Posix.execvp(opt_devices[0], opt_devices);
-    exit_error("Cannot run program %s: %s", opt_devices[0], strerror(errno));
-    return 0;
+    int status;
+    try {
+        Process.spawn_async (null, opt_devices, null,
```

Comment:
Nice! This should even address issue #96, as now the parent pid is always guaranteed to get a SIGINT and can cleanly close its recording, right?

Reply:
Yep, that becomes a non-issue as ioctl recording is handled in the parent pid.

For scripts it'll still be the case for now though. As those have not been moved at this point.